bika/lims/controlpanel/bika_analysisservices.py

```diff
@@ -165,82 +183,72 @@ def __init__(self, context, request):
             self.expand_all_categories = False
             self.ajax_categories = True
             self.category_index = "getCategoryTitle"
 
-        self.columns = {
-            "Title": {
+        self.columns = collections.OrderedDict((
+            ("Title", {
                 "title": _("Service"),
                 "index": "sortable_title",
                 "replace_url": "absolute_url",
-                "sortable": not self.do_cats,
-            },
-            "Keyword": {
+                "sortable": self.can_sort}),
+            ("Keyword", {
                 "title": _("Keyword"),
                 "index": "getKeyword",
                 "attr": "getKeyword",
-                "sortable": not self.do_cats,
-            },
-            "Category": {
+                "sortable": self.can_sort}),
+            ("Category", {
                 "title": _("Category"),
                 "attr": "getCategoryTitle",
-                "sortable": not self.do_cats,
-            },
-            "Methods": {
+                "sortable": self.can_sort}),
+            ("Methods", {
                 "title": _("Methods"),
-                "sortable": not self.do_cats,
-            },
-            "Department": {
+                "sortable": self.can_sort}),
+            ("Department", {
                 "title": _("Department"),
                 "toggle": False,
                 "attr": "getDepartment.Title",
-                "sortable": not self.do_cats,
-            },
-            "Unit": {
+                "sortable": self.can_sort}),
+            ("Unit", {
                 "title": _("Unit"),
                 "attr": "getUnit",
-                "sortable": False,
-            },
-            "Price": {
+                "sortable": False}),
+            ("Price", {
                 "title": _("Price"),
-                "sortable": not self.do_cats,
-            },
-            "MaxTimeAllowed": {
+                "sortable": self.can_sort}),
+            ("MaxTimeAllowed", {
                 "title": _("Max Time"),
                 "toggle": False,
-                "sortable": not self.do_cats,
-            },
-            "DuplicateVariation": {
+                "sortable": self.can_sort}),
+            ("DuplicateVariation", {
                 "title": _("Dup Var"),
                 "toggle": False,
-                "sortable": False,
-             },
-            "Calculation": {
+                "sortable": False}),
+            ("Calculation", {
                 "title": _("Calculation"),
-                "sortable": False,
-            },
-            "CommercialID": {
+                "sortable": False}),
+            ("CommercialID", {
                 "title": _("Commercial ID"),
                 "attr": "getCommercialID",
                 "toggle": False,
-                "sortable": not self.do_cats,
-            },
-            "ProtocolID": {
+                "sortable": self.can_sort}),
+            ("ProtocolID", {
```

Comment:
Column "Protocol ID" is not needed, as well as the field from the content type (in fact, this is one of the reasons why Methods exist)